pinot-controller/src/main/java/org/apache/pinot/controller/api/ControllerAdminApiApplication.java

```diff
@@ -149,4 +140,20 @@ public void filter(ContainerRequestContext containerRequestContext,
       }
     }
   }
+  public class SafeStaticHttpHandler extends CLStaticHttpHandler {
```

Comment:
Please reformat the changes following [Pinot Style](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#set-up-ide)

Reply:
https://github.com/apache/pinot/pull/13124/files

contributed to this PR . closing this